CognitoSyncProject/Assets/DemoScene/SyncCharacterSelection/SaveManager.cs

```diff
@@ -80,7 +83,7 @@ public void InitWithoutFacebook()
         // DefaultCognitoSyncManager is a high level CognitoSync Client which handles all Sync operations at a Dataset level.
         // Additionally, it also provides local storage of the Datasets which can be later Synchronized with the cloud(CognitoSync service)
         // This feature allows the user to continue working w/o internet access and sync with CognitoSync whenever possible
-        syncManager = new DefaultCognitoSyncManager (credentials, new AmazonCognitoSyncConfig { RegionEndpoint = RegionEndpoint.USEast1 });
+		syncManager = new DefaultCognitoSyncManager (credentials, new AmazonCognitoSyncConfig { RegionEndpoint = AmazonInitializer.CognitoRegionEndpoint });
```

Comment:
We already caught this issue and should be fixed when we update the sample. Thanks for noticing it!